test/stateOfDevOpsDashboardHandler.test.ts

```diff
@@ -1,23 +1,26 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
 "use strict";
-import awsSdk from "aws-sdk";
+import * as AWSMock from "aws-sdk-mock";
+import awsSdk, { SSM } from "aws-sdk";
 import LambdaTester from "lambda-tester";
 import chai from "chai";
 import sinonChai from "sinon-chai";
 chai.use(sinonChai);
 const expect = chai.expect;
 import sinon from "sinon";
+import { getAppNamesFromSSM } from "../src/utils";
```

Comment:
You don't need to include underlying function here

Reply:
Agreed. Removed